lib/Chart.svelte

```diff
@@ -36,6 +36,14 @@
     $: publishData = data.publishData;
     $: locale = data.visJSON.locale;
 
+    $: ariaDescription = purifyHtml(
+        get(
+            chart,
+            'metadata.describe.aria-description',
+            '<a><span><b><br><br/><i><strong><sup><sub><strike><u><em><tt><table><thead><tbody><tfoot><caption><colgroup><col><tr><td><th>'
```

Comment:
i think this is supposed to be the whitelist and note the default value?

Reply:
indeed